frontend/src/main/pages/PlayPage.js

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import { Container, CardGroup } from "react-bootstrap";
 import { useParams } from "react-router-dom";
-import { toast } from "react-toastify";
```

Comment:
is there a reason why toast was removed :(

Reply:
We are no longer using the toast in this file. The toast is only used on success, I felt like this was unnecessary because if the operation was successful the user would see the values change. Furthermore these success toasts are really annoying when buying multiple cows.